docs/sources/live/configuration.md

```diff
@@ -0,0 +1,113 @@
++++
+title = "Configuration"
+description = "Grafana Live configuration guide"
+keywords = ["Grafana", "live", "guide", "websocket"]
+weight = 120
++++
+
+# Grafana Live configuration guide
+
+Grafana Live is enabled by default, but as of Grafana v8 it has a strict default for maximum number of connections per Grafana server instance. 
+
+## Max number of connections
+
+Grafana Live uses persistent connections (WebSocket at the moment) to deliver real-time updates to clients. You may need to tune your server infrastructure a bit.
+
+For this reason the number of maximum connections users can establish with Grafana is limited to 100:
+
+```
+[live]
+# max_connections to Grafana Live WebSocket endpoint per Grafana server instance. See Grafana Live docs if you are
+# going to make it higher than default 100 since this can require some OS and infrastructure tuning.
+max_connections = 100
+```
+
+Before making this limit higher make sure your server and infrastructure allow handling more connections. Below we collected a number of common problems which could happen when managing persistent connections, in particular WebSocket connections.
```

Comment:
```suggestion
In case you want to increase this limit, ensure that your server and infrastructure allow handling more connections. The following sections discuss several common problems which could happen when managing persistent connections, in particular WebSocket connections.
```